webpack.config.cjs

```diff
@@ -172,3 +179,69 @@ function extensionModules( mode, target ) {
 		},
 	];
 }
+
+// Create a custom plugin to emit the merged JSON file
+class EmitMergedJsonPlugin {
```

Comment:
I think it would make sense to name this something specific, as it's not intended as a generic JSON-merging plugin, but it specifically generates the schema, and places it in the correct directories. Just a detail.

Something like:

```suggestion
class EmitSubjectsSchemaPlugin {
```